sympy/matrices/matrices.py

```diff
@@ -992,14 +992,14 @@ def rowspace(self, simplify=False):
     @classmethod
     def orthogonalize(cls, *vecs, **kwargs):
         """Apply the Gram-Schmidt orthogonalization procedure
-        to vectors supplied in `vecs`.
+        to vectors supplied in `vecs` .
 
         Arguments
         =========
 
         vecs : vectors to be made orthogonal
-        normalize : bool. Whether the returned vectors
-                    should be renormalized to be unit vectors.
+        normalize : bool. Whether the returned vectors should be renormalized
```

Comment:
I would recommend something like this:

    normalize : bool
        If true, return an orthonormal basis.